app/src/components/AudioTimeline/AudioTrackEditor.tsx

```
@@ -0,0 +1,796 @@
+                className={cn(
+                  'absolute bottom-0 top-0 z-30 w-1 rounded-full bg-accent',
+                  isDraggingPlayhead ? 'cursor-grabbing' : 'cursor-grab',
+                )}
+                style={{ left: `${msToPixels(currentTimeMs)}px` }}
+                onMouseDown={handlePlayheadMouseDown}
+                role="slider"
+                aria-label="Timeline playhead"
+                aria-valuemin={0}
+                aria-valuemax={Math.round(totalDurationMs)}
+                aria-valuenow={Math.round(currentTimeMs)}
+              >
```

Biome (2.4.15)

[error] The HTML element with the interactive role "slider" is not focusable.

(lint/a11y/useFocusableInteractive)